browserslist

```diff
@@ -4,3 +4,4 @@
 
 > 4%
 Last 1 version
+Safari 10
```

Comment:
Is there any benefit to making this `Safari > 9` or similar, to prevent any future weird "gaps" where fixes are included for Safari 10 but not 11?

Reply:
Would `Last 3 Safari versions` do what we want it to?

```
Safari 11.1
Safari 11
Safari 10.1
```

Then it would require less maintenance, because it would automatically update when new versions are released.

Another thing that might be worth looking into is [`browserslist-ga`](https://github.com/browserslist/browserslist-ga). What do you think?

Reply:
I do not have a strong opinion on how it's done as long as the version gap problem is solved.